### DISCUSSION TITLE
why data set is taken as .h5 type instead spectral data

### DISCUSSION THREAD

**Discussion by renjith-vs** [ALebrun-108/BoxSERS#4] [Q&A] [answered]
Hi, this is brilliant work, we need some clarification on the .h5 dataset, tell us about the methodology in data frame conversion from raw spectral data. Anticipating your positive responses.

**Comment by ALebrun-108** [comment, marked as answer]
Hi there, 

This is an excellent question and I would like to thank you for your appreciation. To promote a more fruitful discussion, I would like to invite you to continue this discussion on the issue channel if you would like to. 

Thank you, 

Alexis Lebrun
